content/efcore/getting-started/understanding-data-models.md

```diff
@@ -0,0 +1,48 @@
+# Understanding Data Models 
+ 
+In this lesson, we are going to learn what a data model is and how to manually define a data model for EF Core. 
+ 
+## What is a Data Model? 
+ 
+A data model is a representation of the different types of information within a system and their relationships. For example, in a relational database, the database schema (tables, columns, etc.) would make up the data model. You sometimes hear these data models simply referred to as "models" in EF Core.  
+ 
+## Defining a Data Model 
+ 
+In EF Core, a data model is defined using "plain-old" C# object (POCO) classes. These are classes that solely consist of getters and setters. In EF Core, these POCO classes are referred to as **entities** or **entity classes**. 
+ 
+Let's look at an example. Imagine we are creating a library application. What types of information would be in such a system? Books and readers are two good examples. To include these types of information in our data model, we first define an entity for each of them like so: 
+ 
+```{C#} 
+public class Book 
+{ 
+	public int Id { get; set;} 
```

Comment:
Ok, that's fine then. It's just annoying how the github diff doesn't display tabs